sinfonier-web/app/views/Modules/module.html

```diff
@@ -98,4 +98,27 @@ <h1 id="module-h1-title">&{'Modules.details.title'} '${module?.getName()} (vc${m
 
     #{script 'util/code.js'/}
     #{script 'modules.js'/}
+

```

Comment:
The problem is that I need the module's id and it is rendered in the template, so, pass it to js file it unclean because I should have to used the window var., because of that, I think this is better and cleaner this way.